azure/packages/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -164,4 +178,95 @@ describe("AzureClient", () => {
         // eslint-disable-next-line require-atomic-updates
         console.error = consoleErrorFn;
     });
+
+    /**
+     * Scenario: Test if AzureClient starts the container in write mode. AzureClient will attempt
+     * to start the connection in write mode, but if the write permission is not available or the file is read
+     * only, mode will be read.
+     *
+     * Expected behavior: The container should start with the connectionMode in `write`.
+     */
+    it("can create a container in write mode", async () => {
+        const { container } = await client.createContainer(schema);
+        const containerId = await container.attach();
+        await timeoutPromise((resolve) => container.once("connected", resolve), {
+            durationMs: 1000,
+            errorMsg: "container connect() timeout",
+        });
+
+        const { container: containerGet } = await client.getContainer(containerId, schema);
```

Comment:
Not sure you need this variation. Your two cases below are testing read vs write permissions?

Reply:
removed test case